pinot-common/src/main/java/org/apache/pinot/common/metrics/AbstractMetrics.java

```diff
@@ -58,6 +57,9 @@
 
   private final Class _clazz;
 
+  // The purpose of having _gaugeValues is to make gauge metric updatable.
```

Comment:
Is the comment in line 44 relevant anymore?
/**
 * Common code for metrics implementations.
 * TODO: 1. With gauge updatable, we can remove _gaugeValues 2. Remove methods with callback in name since the callback
 *   function can not be updated.
 */

Reply:
good catch, removed.